.github/workflows/testworkflow.yml

```diff
@@ -0,0 +1,60 @@
+name: testworkflow
+
+on: # run any PRs and main branch changes
+  pull_request:
+  push:
+    branches:
+      - main
+      - 'releases/*'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+
+  testworkflow:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: echo 'https://github.com/fair-software/howfairis' > urls.txt
+      - uses: tortellini-tools/action@main
+        with:
+          repositories: urls.txt
+
+      - run: ls -l .tortellini/out/fair-software/howfairis/gl-license-scanning-report.json
+     
+      - run: |
+          git clone --depth 1 https://github.com/oss-review-toolkit/ort.git /tmp/ort
+          DOCKER_BUILDKIT=1 docker build -t ort /tmp/ort
```

Comment:
probably easier and more reliable for comparison purposes to pull phillipssoftware/ort

Reply:
Done. This made the test much faster. Thanks! However, I got this error message: [This declaration is experimental and its usage must be marked with '@kotlin.ExperimentalStdlibApi' or '@OptIn(kotlin.ExperimentalStdlibApi::class)'](https://github.com/tortellini-tools/action/runs/2819797346?check_suite_focus=true). I could not find out what it meant so I have emptied the file `rules.kts` to get rid of the error.

Reply:
I didnt look into this but I expect there is a method somewhere in the kotlin rules file that we could annotate with a decorator `@something`. I expect that emptying the rules files will lead to misclassifications.

Reply:
Switching to the configuration files used in [testing.yml](https://github.com/tortellini-tools/action/blob/main/.github/workflows/testing.yml) solved the problem. Previously I was using the configuration files used in [https://github.com/tortellini-tools/action/blob/rsd-software-vs-ort/ort/prep.sh](https://github.com/tortellini-tools/action/blob/rsd-software-vs-ort/ort/prep.sh), so that file could probably benefit from an update.